bot/code_review_bot/tasks/clang_tidy_external.py

```diff
@@ -160,7 +170,8 @@ def parse_issues(self, artifacts, revision):
                 if "reliability" in warning
                 else Reliability.Unknown,
                 reason=warning.get("reason"),
-                publish=warning.get("publish"),
+                publish=warning.get("publish")
+                and not warning["flag"].startswith("clang-diagnostic"),
```

Comment:
What about doing `warning["flag"].startswith("mozilla-civet-")` instead of `not warning["flag"].startswith("clang-diagnostic")`?